src/pyclvm/instance/ls.py

```diff
@@ -30,20 +28,20 @@ def ls(**kwargs: str) -> None:
         None
 
     """
-    instances = Ec2InstanceMapping(get_session(kwargs))
+    instances = Ec2AllInstancesData(**kwargs)
     sts_client = boto3.client("sts")
 
     account = sts_client.get_caller_identity().Account
     table = Table(title=f"{account} Account EC2 Instances")
     for column in _COLUMNS:
         table.add_column(column, justify="left", no_wrap=True)
 
-    for name, instance in instances.items():
+    for instance_ in instances:
```

Comment:
rather put the underscore before and not after

also suggest using in the for loop:
id, name, state_value, state_name = _instance

its another operation to execute but is more informative